src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/slurm/SlurmCommandParser.groovy

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2017 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy.execution.jobs.cluster.slurm
+
+
+import de.dkfz.roddy.config.ResourceSet
+import de.dkfz.roddy.execution.jobs.BEJobID
+import de.dkfz.roddy.execution.jobs.GenericJobInfo
+import de.dkfz.roddy.tools.BufferUnit
+import de.dkfz.roddy.tools.BufferValue
+import de.dkfz.roddy.tools.ComplexLine
+import de.dkfz.roddy.tools.TimeUnit
+import groovy.transform.CompileStatic
+
+import static de.dkfz.roddy.StringConstants.SPLIT_COMMA
+import static de.dkfz.roddy.StringConstants.SPLIT_EQUALS
+
+/**
+ * Used to convert commands from cli to e.g. GenericJobInfo
+ *
+ * TODO It needs further work, in case it is needed
+ */
+@CompileStatic
+class SlurmCommandParser {

```

Comment:
Lots of copy/paste (bad) code. Alas, I am not sure whether it is worth doing this better than you did.